c2corg_api/views/article.py

```diff
@@ -57,6 +62,24 @@ def collection_post(self):
                 validate_article_update,
                 validate_associations_update])
     def put(self):
+        if not self.request.has_permission('moderator'):
+            article_id = self.request.validated['id']
+            article = DBSession.query(Article).get(article_id)
+            if article is None:
+                raise HTTPNotFound('No article found for id %d' % article_id)
```

Comment:
Ok, sometimes yes, sometimes no - like [here for image](https://github.com/c2corg/v6_api/blob/master/c2corg_api/views/image.py#L144). I will change both now but there are still around +-50 occurrences of "...' % ..." elsewhere. I am not sure if they all can be just simply changed to `.format` instead